cmd/collection/main.go

```diff
@@ -373,7 +374,13 @@ func main() {
 		}).
 		// Epoch manager encapsulates and manages epoch-dependent engines as we
 		// transition between epochs
-		Component("epoch manager", func(_ cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+		Component("epoch manager", func(nodeBuilder cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+
+			workerPool := workerpool.New(4)
+			nodeBuilder.Component("epoch factory worker pool", func(_ cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
+				return workerPool, nil
+			})
```

Comment:
Better move this close to [where it's used](https://github.com/onflow/flow-go/pull/1679/files#diff-4688e4f1e6bc5063d6bcb143912d69e213eff1019cf5caed6f75218349b70bc2R456-R465).

```suggestion
			workerPool := workerpool.New(4)
			nodeBuilder.Component("hotstuff vote processer worker pool", func(_ cmd.NodeBuilder, node *cmd.NodeConfig) (module.ReadyDoneAware, error) {
				return workerPool, nil
			})
```